NotifySwiftQuickstart/AppDelegate.swift

```diff
@@ -29,6 +29,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         UIApplication.shared.registerUserNotificationSettings(settings)
         UIApplication.shared.registerForRemoteNotifications()
     }
+    
+    self.environment = ProvisioningProfileInspector().apnsEnvironment()
+    let envString = self.environment == APNSEnvironment.development ? "Development" : "Production"
```

Comment:
Also, this ternary statement doesn't account for unknown - so if there isn't a provisioning profile, it will print out Production, which is confusing.